src/main/resources/assets/js/application/recruiter/services/tdprPersonsService.js

```diff
@@ -18,7 +18,12 @@ define(['application/recruiter/tdprRecruiterModule'], function (tdprRecruiterMod
             return $http.put("/api/person/create/", person).then(function (response) {
                 return response;
             }, function (err) {
-                err.message = "Interviewer adding failed.";
+
+                if (err.status == 409) {
```

Comment:
For readability's sake it would be good to create a constants object with HTTP status codes.